### PR TITLE
Travis: Deploy documentations on gh-pages for deployment and master

### PR DESCRIPTION
**Pull request by yukiisbored**
Closes https://github.com/loklak/loklak_server/issues/964

I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only strictly only one commit per issue.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -32,8 +31,18 @@ before_deploy:
...
 - ssh-add .utility/loklakserver.1
 deploy:
-  provider: script
-  skip_cleanup: true
-  script: ssh loklak@104.155.231.103 /opt/loklak/automatic-deploy.sh
-  on:
-    branch: master
+  - provider: script
+    skip_cleanup: true
+    script: ssh loklak@104.155.231.103 /opt/loklak/automatic-deploy.sh
```
Comment (sudheesh001) [inline .travis.yml]: Is this your personal loklak server ?

Comment (mariobehling) [inline .travis.yml]: @yukiisbored Please help to avoid dependencies on outside services and keep deploy scripts in the  repo itself. Thank you.

Comment (yukiisbored) [inline .travis.yml]: @mariobehling Okay, I'll add it on a separate PR.
@sudheesh001 No, that's for test.loklak.org. it was there before this PR

Comment (sudheesh001) [inline .travis.yml]: Cool, didn't know that.

**Pull request merged**